core/src/main/resources/META-INF/resources/primefaces-extensions/clockpicker/1-clockpicker-widget.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * PrimeFaces Extensions ClockPicker Widget.
+ * 
+ */
+PrimeFaces.widget.ExtClockPicker = PrimeFaces.widget.BaseWidget.extend({
+	/**
+     * Initializes the widget.
+     * 
+     * @param {object}
+     *        cfg The widget configuration.
+     */
+    init : function(cfg) {
+        this._super(cfg);
+        this.id = PrimeFaces.escapeClientId(cfg.id);
+        this.cfg = cfg;
+        this.jqEl = this.jqId + '_input';
+        this.jq = $(this.jqEl);
+
+        this.jq.clockpicker(this.cfg);
+        // pfs metadata
+        $(this.jqId + '_input').data(PrimeFaces.CLIENT_ID_DATA, this.id);
+        this.originalValue = this.jq.val();
+        
+    }
```

Comment:
We should add `refresh` and `destroy` methods.

We should also exposed `show(), hide(), toggleView()` widgrt methods from https://github.com/weareoutman/clockpicker?tab=readme-ov-file#operations

Reply:
I included refresh, destroy, show, hide client side API. The toggleView does not work well so I exclude it for now.